README.md

```diff
@@ -17,6 +17,7 @@ developers) scrollbar plugin.
 * Use plain `scrollTop` and `scrollLeft`
 * Scrollbar style is fully customizable
 * Efficient update on layout change
+* SSR friendly, e.g. through [ngx-perfect-scrollbar](https://github.com/zefoy/ngx-perfect-scrollbar)
```

Comment:
Supporting SSR is nice, but I'm afraid I personally do not think it's a selling point of ps. Can you please revert this?

Reply:
Fair enough, it's a selling point to me as since I've started to look into SSR I've realized it's really hard to find SSR friendly libraries :)